packages/dds/cell/src/cell.ts

```diff
@@ -43,6 +44,58 @@ interface ICellValue {
     value: unknown;
 }
 
+interface ICellContent {
```

Comment:
is this used? doesn't look referenced or exported

Reply:
It is used in the function` summarizeCore` and `loadCore`. It includes the attribution instead of storing the cell value only.

Reply:
oh, for some reason thought this was in a different file from the cell implementation. it still seems like this should maybe combine with `ICellValue`--it's attempting to serve a similar purpose, right? and the summary format change you're making is backward-compatible (adds an optional property which is handled gracefully when absent)

Reply:
Yeah true, I combined the `ICellContent` into `ICellValue`